doc/source/index.rst.template

```diff
@@ -1,26 +1,21 @@
 .. pandas documentation master file, created by
 
+.. module:: pandas
+
 *********************************************
 pandas: powerful Python data analysis toolkit
 *********************************************
 
-`PDF Version <pandas.pdf>`__
-
-`Zipped HTML <pandas.zip>`__
-
-.. module:: pandas
-
 **Date**: |today| **Version**: |version|
 
-**Binary Installers:** https://pypi.org/project/pandas
-
-**Source Repository:** https://github.com/pandas-dev/pandas
-
-**Issues & Ideas:** https://github.com/pandas-dev/pandas/issues
-
-**Q&A Support:** https://stackoverflow.com/questions/tagged/pandas
+**Download documentation**: `PDF Version <pandas.pdf>`__ | `Zipped HTML <pandas.zip>`__
 
-**Developer Mailing List:** https://groups.google.com/forum/#!forum/pydata
```

Comment:
Can you leave an entry for "Developer Mailing List", but point it to pandas-dev mailing list?

Reply:
Would you have "Users Mailing List" and "Developers Mailing List"?

I'm -1 on adding `pandas-dev` in the home page, a bit concerned about start getting spam of newbie questions there, because of the confusion on what's pandas developer means (contributor vs user)

Reply:
OK, let's leave that discussion for later. Merging

Reply:
>  a bit concerned about start getting spam of newbie questions there,

That's a risk, but I have the feeling that the mailing list is now so *un*known, that we don't get any spam questions, but also hardly developer related questions

Reply:
That sounds indeed good!